static/scripts/javascript.js

```diff
@@ -539,6 +539,18 @@ var app = {
         var el = e.target.closest(".js-results-container");
         el.setAttribute("data-fetch-url", e.target.value);
         el.dispatchEvent(new Event("update"));
+        if (e.target.id === "find-race") {
+          $('#find-candidate').val('1').trigger('change');
+          $('#find-district').val('1').trigger('change');
+        }
+        else if (e.target.id === "find-candidate") {
+          $('#find-race').val('1').trigger('change');
+          $('#find-district').val('1').trigger('change');
+        }
+        else {
+          $('#find-race').val('1').trigger('change');
+          $('#find-candidate').val('1').trigger('change');
+        };
```

Comment:
Loop through all the `.js-select` boxes. If the box in the loop is not the one causing the event skip it, otherwise tell it to reset to default.

Reply:
Select2.org says use `$('.js-select2').val(null).trigger('change');`